pkg/virt-controller/watch/pool.go

```diff
@@ -1215,9 +1219,15 @@ func (c *PoolController) updateStatus(origPool *poolv1.VirtualMachinePool, vms [
 	}
 
 	pool.Status.Replicas = int32(len(vms))
+	pool.Status.ReadyReplicas = 0
```

Comment:
We don't want to assert the number of ReadyReplicas since `updateStatus` will be called during the pool execution loop.  Instead we need to compare current state to expected state:
1) check the number of ReadyReplicas
2) compare ReadyReplicas to Replicas (return if we are done)
3) update